shard-operator/src/main/java/com/redhat/service/smartevents/shard/operator/providers/CustomerBridgeNamespaceProviderImpl.java

```diff
@@ -20,16 +21,16 @@
 import io.quarkus.scheduler.Scheduled;
 
 @ApplicationScoped
-public class CustomerNamespaceProviderImpl implements CustomerNamespaceProvider {
+public class CustomerBridgeNamespaceProviderImpl implements CustomerBridgeNamespaceProvider {
 
-    private static final Logger LOGGER = LoggerFactory.getLogger(CustomerNamespaceProviderImpl.class);
+    private static final Logger LOGGER = LoggerFactory.getLogger(CustomerBridgeNamespaceProviderImpl.class);
 
     @Inject
     KubernetesClient kubernetesClient;
 
     @Override
-    public Namespace fetchOrCreateCustomerNamespace(String customerId) {
-        final String name = this.resolveName(customerId);
+    public Namespace fetchOrCreateCustomerBridgeNamespace(String customerId, String bridgeId) {
+        final String name = this.resolveName(customerId, bridgeId);
```

Comment:
@manstis We need to consider existing bridges. Based upon this change we won't be able to update/delete existing bridges in our `dev` or `stable` environment as the `namespace` lookup will fail.

What I'd suggest is that we have a namespace look-up order of preference in here, something like:

- Check to see if a namespace exists in the new format, if yes, return it
- Check to see if a namespace exists in the old format, if yes, return it
- If no namespace exists, create it in the new format.

By my thinking that should mean that existing bridges continue to function as expected, with any new bridge being created with the new namespace format.